spec/schemas/_common.yaml

```diff
@@ -1814,39 +1814,39 @@ components:
           description: The total amount of time for all shard scroll operations.
           $ref: '#/components/schemas/Duration'
         scroll_time_in_millis:
-          description: The total amount of time for all shard scroll operations, in milliseconds.
+          description: The total amount of time for all shard scroll operations, in milliseconds. 
           $ref: '#/components/schemas/DurationValueUnitMillis'
         scroll_total:
           description: The total number of shard scroll operations.
           type: number
         point_in_time_total:
-          description: The total number of shard Point in Time (PIT) contexts that have been created (completed and active) since the node last restarted.
+          description: The total number of shard Point in Time (PIT) contexts that have been created (completed and active) since the node last restarted. 
           type: number
         point_in_time_time:
           $ref: '#/components/schemas/Duration'
         point_in_time_time_in_millis:
-          description: The amount of time that shard PIT contexts have been held open since the node last restarted, in milliseconds.
+          description: The amount of time that shard PIT contexts have been held open since the node last restarted, in milliseconds. 
           $ref: '#/components/schemas/DurationValueUnitMillis'
         point_in_time_current:
-          description: The number of shard PIT contexts currently open.
+          description: The number of shard PIT contexts currently open. 
           type: number
         suggest_current:
-          description: The number of shard suggest operations that are currently running.
+          description: The number of shard suggest operations that are currently running. 
           type: number
         suggest_time:
           description: The total amount of time for all shard suggest operations.
           $ref: '#/components/schemas/Duration'
         suggest_time_in_millis:
-          description: The total amount of time for all shard suggest operations, in milliseconds.
+          description: The total amount of time for all shard suggest operations, in milliseconds. 
```

Comment:
"The total amount of time taken to complete all shard suggest operations, in milliseconds"?